react/features/large-video/actions.js

```diff
@@ -145,7 +145,16 @@ function _electParticipantInLargeVideo(state) {
                 //    As a last resort, pick the last participant who joined the
                 //    conference (regardless of whether they are local or
                 //    remote).
-                participant = participants[participants.length - 1];
+                //
+                // HOWEVER: We don't want to show poltergeist or other bot type participants on stage
+                // automatically, because it's misleading (users may think they are already
+                // joined and maybe speaking).
+                for (const p of participants) {
+                    // This loop goes forward, so we don't stop until the end to replicate the original (last
```

Comment:
Why not go backwards and exit early if your intent is to find the last one?

Reply:
Tiggers can't climb down because their tails get in the way